vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -314,7 +314,7 @@ void ReactInstanceWin::Initialize() noexcept {
             auto instanceWrapper = facebook::react::CreateReactInstance(
                 std::string(), // bundleRootPath
                 std::move(cxxModules),
-                nullptr,
+                m_options.TurboModuleProvider,
```

Comment:
When verifying that we are able to run a TurboModule, you may find that they only work when not web-debugging.  I think we will probably need to add code to the instance to load the turbomodules as native modules when web debugging.

Reply:
Now Clipboard only registers to TurboModule when web debugger is disabled